table/static/js/saveAsFile.js

```diff
@@ -1,4 +1,10 @@
 $("#saveAsFile").click(function() {
+  var tmp_attr = $("#course-table").attr("class");
+  $("#course-table").removeAttr("class");
+  $("#course-table").css("width", "200%");
```

Comment:
Well, it won't be modified in this PR.
I'll find another way to create this same effect.